src/shabdawali.js

```diff
@@ -254,7 +254,36 @@ Shabdawali.prototype.typeText = function(cLine){
         }
     }
 }
+/*
+Complexity will have values ranging from 1-4,
+where '1' will have no complexity and the value
+of complexity will increase by one for each of 
+the conditions satisfied
+*/
+Shabdawali.prototype.wordComplexity = function(word){
```

Comment:
It should be a utility method instead of instance method. `var wordComplexity = function(word){`